Assets/HoloToolkit/Build/Editor/BuildDeployTools.cs

```diff
@@ -135,5 +143,47 @@ public static bool BuildAppxFromSolution(string productName, string msBuildVersi
                 return true;
             }
         }
+
+        private static void IncrementPackageVersion()
+        {
+            // Find the manifest, assume the one we want is the first one
+            string[] manifests = Directory.GetFiles(BuildDeployPrefs.AbsoluteBuildDirectory, "Package.appxmanifest", SearchOption.AllDirectories);
+            if (manifests.Length == 0)
+            {
+                Debug.LogError("Unable to find Package.appxmanifest file for build (in path - " + BuildDeployPrefs.AbsoluteBuildDirectory + ")");
+                return;
+            }
+            string manifest = manifests[0];
+
+            XElement rootNode = XElement.Load(manifest);
+            XNamespace ns = rootNode.GetDefaultNamespace();
+            var identityNode = rootNode.Element(ns + "Identity");
+            if (identityNode == null)
+            {
+                Debug.LogError("Package.appxmanifest for build (in path - " + BuildDeployPrefs.AbsoluteBuildDirectory + ") is missing an <Identity /> node");
+                return;
+            }
+
+            // We use XName.Get instead of strint -> XName implicit conversion because
+            // when we pass in the string "Version", the program doesn't find the attribute.
+            // Best guess as to why this happens is that implicit string conversion doesn't set the namespace to empty
+            var versionAttr = identityNode.Attribute(XName.Get("Version"));
+            if (versionAttr == null)
+            {
+                Debug.LogError("Package.appxmanifest for build (in path - " + BuildDeployPrefs.AbsoluteBuildDirectory + ") is missing a version attribute in the <Identity /> node.");
+                return;
+            }
+
+            // Assume package version always has a '.'.
+            // According to https://msdn.microsoft.com/en-us/library/windows/apps/br211441.aspx
+            // Package versions are always of the form Major.Minor.Build.Revision
+            var curVersionStr = versionAttr.Value;
```

Comment:
Nit: I'd use the Version class here to cleanup the manual string manipulation (e.g. Version.TryParse)